sbt-dotty/src/dotty/tools/sbtplugin/DottyIDEPlugin.scala

```diff
@@ -64,6 +63,11 @@ object DottyIDEPlugin extends AutoPlugin {
     Def.derive(projectConfig := {
       if (sources.value.isEmpty) None
       else {
+        // Not needed to generate the config, but this guarantees that the
+        // generated config is usable by an IDE without any extra compilation
+        // step.
+        val _ = compile.value
```

Comment:
OK but then, why is `compileForIDE` still necessary? It doesn't seem to be called by anything.